pac4j-core/src/main/java/org/pac4j/core/exception/MultipleAccountsFoundException.java

```diff
@@ -6,7 +6,7 @@
  * @author Jerome Leleu
  * @since 1.8.0
  */
-public class MultipleAccountsFoundException extends TechnicalException {
+public class MultipleAccountsFoundException extends CredentialsException {
```

Comment:
@victornoel : I made the `MultipleAccountsFoundException` a checked `CredentialsException`, is there any reason to keep it as a `TechnicalException`?